AdminPortal/src/pages/admin.tsx

```diff
@@ -2,15 +2,54 @@
+          </svg>
+        </div>
+        <div >
+          <table className={styles.table}>
+            <thead className={styles.tableheader}>
+              <tr>
+                <th>Name</th>
+                <th>Mobile Number</th>
+                <th>Role</th>
+              </tr>
+            </thead>
+            <tbody className={styles.tablebody}>
+              <tr>
+                <td>Thinley Choden</td>
+                <td>+1 (234) 567-8910</td>
+                <td>Primary Admin</td>
+              </tr>
+              <tr>
+                <td>Carly Shay</td>
+                <td>+1 (234) 567-8910</td>
+                <td>Secondary Admin</td>
+              </tr>
+              <tr>
+                <td>Spencer Shay</td>
+                <td>+1 (234) 567-8910</td>
+                <td>Secondary Admin</td>
+              </tr>
+              <tr>
+                <td>Freddie Benson</td>
+                <td>+1 (234) 567-8910</td>
+                <td>Secondary Admin</td>
+              </tr>
```

Comment:
Just for future use, can we have it generate these rows and cells given a list instead of hard coding it? We'll be using this later when we pull admin users from the backend